.release-it.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    git: {
+        commitMessage: 'chore: release ${version}',
+        tagName: 'v${version}',
+    },
+    npm: {
+        publish: true,
+    },
+    github: {
+        release: true,
+    },
+    plugins: {
+        '@release-it/conventional-changelog': {
+            preset: 'angular',
```

Comment:
maybe `conventionalcommits` instead of `angular`? 🤔

Reply:
yeah, good point, [release-it provides a few different configurations](https://github.com/release-it/conventional-changelog#preset ) so I will change it to `conventionalcommits`